doc/extensions.md

```diff
@@ -0,0 +1,18 @@
+# Extending Files
+
+Here you will find information on how you can tune the behaviour of certain parts
+of the library.
+
+## Disabling file existance checks in the FileManager during file loading
+
+Although in the majority of cases you will want the `FileManager` to check if
+the loaded files actually exist in their defined filesystems, there may be cases
+where that is unnecessary. For example: generating a financial report using entities
+that also have invoice scans that are irrelevant to the report itself. In this
+case the file checks not only slow down the process significantly, but can break
+it altogether if for any reason a file has gone missing.
+
+In order to disable these checks, you can use the `FSi\Component\Files\FileManagerConfigurator\FileExistanceChecksConfigurator`
+interface. Just check if the implementation of the `FileManager` that you are using
```

Comment:
please add a service alias `FSi\Component\Files\FileManagerConfigurator\FileExistanceChecksConfigurator` on the current `FileManager` implementation and advice / encourage user to use the new "service" for changing configuration rather than checking if `FileManager` implements the new interface